CHANGES.md

```diff
@@ -78,32 +92,28 @@
   see [this table](https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/24.0.0/artifact_details.html).
 * Removed avro-python3 dependency in AvroIO. Fastavro has already been our Avro library of choice on Python 3. Boolean use_fastavro is left for api compatibility, but will have no effect.([BEAM-13016](https://github.com/apache/beam/pull/15900)).
 * MultiMap side inputs are now supported by the Go SDK ([BEAM-3293](https://issues.apache.org/jira/browse/BEAM-3293)).
+* Remote packages can now be downloaded from locations supported by apache_beam.io.filesystems. The files will be downloaded on Stager and uploaded to staging location. For more information, see [BEAM-11275](https://issues.apache.org/jira/browse/BEAM-11275)
+* Added support for cloudpickle as a pickling library for Python SDK. To use cloudpickle set pipeline option: --pickler_lib=cloudpickle
```

Comment:
link BEAM-8123

Reply:
cloudpickle is not yet in 2.35.0, it should be a part of 2.36.0

Reply:
will fix that, thanks